cloudfoundry-community/stackdriver-tools
# Version the stackdriver-nozzle from the BOSH release semver

On CI builds the version.Release() and UserAgent will have the same version as the BOSH release.

Validation:
- Build (`make build`) with no `src/stackdriver-nozzle/release` file, observe version is `dev` 
- Add the release file (`echo 0.0.1 > src/stackdriver-nozzle/release`), build, observe version is `0.0.1`

Possible issues:
- I haven't tested the CI code
- We now need to force create the bosh release. I think that's ok because we are expecting the discrepancy and the check for local files is for developers and not CI.
